package.json

```diff
@@ -37,6 +37,7 @@
         "jwt-decode": "^3.1.2",
         "lodash": "^4.17.21",
         "lowdb": "^1.0.0",
+        "luxon": "^1.26.0",
         "moment": "^2.29.1",
```

Comment:
do we still need moment?

Reply:
We still use it to check if the access token has expired or not. Could rewrite these parts to use luxon instead and thereby remove the moment dependency. I decided not to do it now, but if you prefer it I don't mind.

Reply:
One less dependency is always great. Will you do it?